CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,72 @@
+
+The following behaviors are expected and requested of all devroom participants:
+
+- Be respectful and value each other’s ideas, styles and viewpoints.
+- Be considerate of others' participation, including use of conference time
+- Be inclusive and help new perspectives be heard
+- Be open to learning from others
+- Be mindful of your surroundings and of your fellow participants. Please alert the devroom organisers if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
+
+As an overriding general rule, please be intentional in your actions and humble in your mistakes.
+
+We do not tolerate harassment of conference participants in any form – both online and at the in-person event. Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, parties, Twitter, Mastodon and other online media. 
+
+Unacceptable behaviors include the following:
+- Intimidating, harassing, lewd, demeaning, bullying, stalking, trolling or threatening speech or actions
+- Unwelcome sexual attention or physical contact
+- Any real or implied threat of physical harm
+- Sustained disruption of speakers or events (verbally or physically)
+- Publishing others’ private information, such as a physical or email address, without their explicit permission
+- Advocating or encouraging any of the above behaviors
+
+For more information and examples about what constitutes harassment, please refer to OpenCon’s Code of Conduct in Brief and the [Gathering for Open Source Hardware’s](https://openhardware.science/gosh-2017/gosh-code-of-conduct/) examples of behaviour.
+
+Conference participants violating these rules may be asked to leave or removed from the Open Research devroom (virtual or in-person) at the discretion of the organisers. 
+
+# Enforcement 
+Participants asked to stop any harassing behavior are expected to comply immediately.
+
+If a participant engages in harassing behavior, the conference organisers may take any action they deem appropriate, including warning the offender or expulsion from the online or in-person conference room. 
+
+The organising team may take action to redress anything designed to disrupt, or with the clear impact of disrupting, the event or making the environment hostile for any participants.
+
+# Reporting
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of the devroom organising team immediately. 
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may also be reported to the organisers responsible for enforcement at [anonymous Google form?]
```

Comment:
```suggestion
Instances of abusive, harassing, or otherwise unacceptable behavior may also be reported to the organisers responsible for enforcement at [framaform linked here]. 
```